frontend/src/api/routes/robots/robotsRetrieval.js

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
I would rename the file to just "robots" like on the server side. Same goes for rpaParameterRetrieval and rpaAttributesRetrieval

Reply:
is fixed